Engine/src/Engine/Application.cpp

```diff
@@ -1,9 +1,13 @@
 #include "Application.h"
 
+#include "Entity.h"
+#include "Scene.h"
+
 namespace Engine
 {
 	void Application::Run()
 	{
-		while (true);
+		Scene scene;
```

Comment:
If these lines are here for testing please discard the changes to this file

Reply:
done